gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectAttributes.java

```diff
@@ -101,7 +137,7 @@ public FileTime lastModifiedTime() {
    */
   @Override
   public ImmutableMap<String, String> userMetadata() {
-    return ImmutableMap.copyOf(metadata.getOptions().getUserMetadata());
+    return ImmutableMap.copyOf(info.metadata());
```

Comment:
Changed it to deal with null info. I'm copying into an ImmutableMap for type reasons, so I can't just remove it (sadly).